pom.xml

```diff
@@ -309,6 +309,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <jgrapht.version>1.0.1</jgrapht.version>
 
     <vault.driver.version>5.1.0</vault.driver.version>
+    <java.uuid.generator.version>3.1.4</java.uuid.generator.version>
```

Comment:
Can you please explain how this specific version was selected?

Reply:
as the first suggestion in IDE, changed to 4.3.0 (the last released version from the Maven Central)